kie-wb-common-react/boxed-expression-editor/src/components/Resizer/dom/Cell.ts

```diff
@@ -66,13 +66,54 @@ export class Cell {
       return;
     }
 
-    const parentRect = this.getParentRow()?.getBoundingClientRect();
+    const parentRect = this.getParentRow()!.getBoundingClientRect();
 
     if (parentRect) {
       this.setWidth(Math.round(parentRect.right) - Math.round(this.getRect().x) - BORDER);
     }
   }
 
+  refreshWidthAsLastGroupColumn(): void {
+    if (!this.isColSpanHeader()) {
+      return;
+    }
+
+    const refSibling = this.getParent()?.parentElement?.nextSibling;
+
+    if (!refSibling) {
+      return;
+    }
+
+    const children = [].slice.call((refSibling as HTMLElement).querySelectorAll(`.${this.getHeaderType()}`));
+    const childrenRects = children.map((c: HTMLElement) => c.getBoundingClientRect());
+    const x = Math.min(...childrenRects.map((c: DOMRect) => c.x));
+    const right = Math.max(...childrenRects.map((c: DOMRect) => c.right));
+
+    this.setWidth(right - x - BORDER * 2);
+  }
+
+  isColSpanHeader(): boolean {
+    return this.getParent()?.classList.contains("colspan-header") || false;
+  }
+
+  private getHeaderType() {
```

Comment:
on other plae in this PR, I saw we have also some "output-" as default column prefix. As I do not see it here, shouldn't we:
- add it also here, or
- remove it from whole PR?

Reply:
Let me give you a bit of context about this topic:
Currently `output-` is the default column prefix for the `output` columns type.
But, in this method of `Cell.ts` we are checking in the CSS classes for the presence of the column type, that can be `input`, `output` or `annotation` independently from the name of the columns (that user can rename).

Reply:
ok, thank you for an explanation